src/cmd-compress

```diff
@@ -117,10 +114,7 @@ def compress_one_builddir(builddir):
         img = buildmeta['images'][img_format]
```

Comment:
I think this is harmless, but I wonder if we still want this `img` declaration here. Before this commit `img` wasn't a loop variable and now it is.

Reply:
Yeah, this is harmless but redundant. We can either merge as is, or cherry-pick 97c522781cb3a2ed6fb4755450292bbab819317e before c2458977cdbceb0b95973798ae99cdaa26de0fb8 and that *should* also fix the merge conflict I had to manually resolve..